src/controllers/sign.out.controller.ts

```diff
@@ -14,7 +13,7 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
     return res.render(config.SIGN_OUT_PAGE, {
       previousPage: previousPageUrl,
       url: config.REGISTER_AN_OVERSEAS_ENTITY_URL,
-      saveAndResume: isActiveFeature(config.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022),
+      saveAndResume: true,
```

Comment:
Guessing we can't remove this (and the other associated things I've seen) until the Update save & resume flag is removed?

Reply:
Yes, we can't remove this yet because of this update flag.